requirements.txt

```diff
@@ -1,2 +1,3 @@
 fastapi >= 0.104.1
-uvicorn >= 0.24.0
\ No newline at end of file
+uvicorn >= 0.24.0
+sunpy[all]
```

Comment:
probably shouldn't need sunpy[all], but we have similar issues on the other repo.

Reply:
prob only need `[net]` but need to test

Reply:
Done